ha/setup/ha_setup.py

```diff
@@ -291,8 +296,14 @@ def process(self):
                         self._confstore.set(f"{const.CLUSTER_CONFSTORE_NODES_KEY}/{node_name}")
                         Log.info(f"Added new node: {node_name}")
                         node_added = True
+                        nodes = self._confstore.get(f"{const.CLUSTER_CONFSTORE_NODES_KEY}")
+                        if ConfigCmd.DEV_CHECK == False and node_added == True and len(nodes.keys()) == 2:
+                            mgmt_info = self._get_mgmt_vip(machine_id, cluster_id)
+                            mgmt_vip(f"{const.RA_LOG_DIR}/cortx-cib_mgmt.xml", create=True, push=True, mgmt_info=mgmt_info)
+                            Log.info(f"Added management vip to cluster {mgmt_info}")
                         break
                     except Exception as e:
+                        node_added = False
```

Comment:
@ajay-paratmandali Do you think we need to delete the mgmt VIP resource if the node addition fails for some reason?

Reply:
if the node addition fails then vip creation will not be executed 

mgmt_vip()
is hadling error, if vip detail is invalid then vip will not be created
So not need to delete resource